packages/web-components/fast-foundation/src/search/search.ts

```diff
@@ -27,8 +27,8 @@ export type SearchOptions = StartEndOptions;
  * @slot clear-button - The clear button
  * @slot clear-glyph - The clear glyph
  * @csspart label - The label
- * @csspart root - The element wrapping the control, including start and end slots
- * @csspart control - The element representing the input
+ * @csspart control - The element wrapping the control, including start and end slots
```

Comment:
Nit, "the element wrapping the control" doesn't make sense if this _is_ the control :)